pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -1191,6 +1238,53 @@ func (nc *Controller) ComputeZoneState(nodeReadyConditions []*v1.NodeCondition)
 	}
 }
 
+// reconcileNodeLabels reconciles node labels.
+func (nc *Controller) reconcileNodeLables(nodeName string) error {
```

Comment:
nit: typo

Reply:
What a wonderfully consistent typo that I made everywhere.